config/routes.rb

```diff
@@ -18,32 +15,34 @@
   resources :orders
 
   namespace :admin do
+    get 'dashboard/index'
     get '', to: 'dashboard#index', as: '/'
     get 'affiliate/:id/show_affiliate_workers',
         to: 'affiliates#show_affiliate_workers',
         as: :show_affiliate_workers
     resources :users
     resources :affiliates
-    resources :taxes
+    resources :taxes do
+      get 'delete',
+          to: 'taxes#delete',
+          as: :delete
+    end
     resources :cars
     resources :user, controller: 'admin/users'
     resources :order_statuses, only: [:index]
     resources :orders do
-      resources :invoices
+      resources :invoices, only: %i[new create edit update]
       resource :car_assignment, only: %i[new create] do
-      get 'edit',
-          to: 'car_assignment#edit',
-          as: :edit
-      post 'update',
-               to: 'car_assignment#update',
-               as: :update
+        get 'edit',
```

Comment:
Why so many CRUD actions (edit, update) already added by routing DSL ? Maybe you just need to change `only: %i[new create]` or configure `resource :car_assignment` (if it doesn't see some actions, controller or anything else) ? :)